### PR TITLE
Fix/address docs review

### PR DESCRIPTION
**Pull request by adamsilverstein**
### Issue Number

Address docs feedback.

https://github.com/GoogleChromeLabs/wp-sitemaps/issues/164#issuecomment-625523450

### Description
A summary of the change. The summary should include what problem is addressed and how it has been fixed.

### Screenshots (before and after if applicable)
If your PR includes visual changes include before and after screenshots showing the change.

### Type of change
Please select the relevant options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Enhancement (change which improves an existing feature. E.g., performance improvement, docs update, etc.)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Steps to test
Describe the tests required to verify your changes.
Provide instructions so the PR Tester can check functionality and also list any relevant details and / or dependencies required for your tests.

### Acceptance criteria
- [x] My code follows WordPress coding standards.
- [x] I have performed a self-review of my own code.
- [x] If the changes are visual, I have cross browser / device tested.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added test instructions that prove my fix is effective or that my feature works.

### CODE REVIEW

File: tests/phpunit/sitemaps.php [hunk 1]
```diff
@@ -372,9 +372,9 @@ public function test_get_url_list_cpt_private() {
 
 	/**
 	 * Helper function for building an expected url list.
-	 *
 	 * @param string $type An object sub type, e.g., post type.
-	 * @param array  $ids  An array of object IDs.
+	 *
```
Comment (swissspidy) [inline tests/phpunit/sitemaps.php:375]: Errant space?

**Comment by swissspidy** [comment]
@adamsilverstein Looks like merging Felix's refactor PR caused tons of conflicts here. Sorry about that! Let me know if you'd need help.

**Comment by adamsilverstein** [comment]
> @adamsilverstein Looks like merging Felix's refactor PR caused tons of conflicts here. Sorry about that! Let me know if you'd need help.

No worries, I will review and re-apply my changes.

File: tests/phpunit/sitemaps.php (hunk 1 above)
Comment (adamsilverstein) [reply tests/phpunit/sitemaps.php:375]: removed.

**Comment by adamsilverstein** [comment]
@swissspidy I resolved the merge conflicts and checked that only my docs changes appear in the PR. This is ready for review.

**Review by swissspidy — APPROVED** [review]

**Pull request merged**